_includes/configure-registry-json.md

```diff
@@ -8,14 +8,16 @@ least one organization the user is a member of, they can sign in to Docker
 Desktop and access all their organizations.
 
 Based on the user's operating system, you must create a `registry.json` file at the following location and make sure the file can't be edited by the user:
-   - Windows: `/ProgramData/DockerDesktop/registry.json`
-   - Mac: `/Library/Application Support/com.docker.docker/registry.json`
+
+- Windows: `/ProgramData/DockerDesktop/registry.json`
+- Mac: `/Library/Application Support/com.docker.docker/registry.json`
+- Linux: `/usr/share/docker-desktop/registry/registry.json`
```

Comment:
Wondering if it would be nice to make this a table 🤔 

```suggestion
| Platform | Location                                                       |
|----------|----------------------------------------------------------------|
| Windows  | `/ProgramData/DockerDesktop/registry.json`                     |
| Mac      | `/Library/Application Support/com.docker.docker/registry.json` |
| Linux    | `/usr/share/docker-desktop/registry/registry.json`             |
```

<img width="1006" alt="Screenshot 2022-08-27 at 00 00 26" src="https://user-images.githubusercontent.com/1804568/186996334-d9c11825-faed-473f-b6a3-0a172df19945.png">

Reply:
This looks nice @thaJeztah  and easier to distinguish each Platform.